test/blueprints/Do.Test.Blueprints.Service/Authentication/Authentication.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Do.Test;
+
+public class Authentication(TimeProvider _timeProvider)
```

Comment:
this class has conflict with its namespace/folder, never make them the same. need to find another name for this class